### PR TITLE
docs: update incorrect batch size reference

### PR DESCRIPTION
**Pull request by RahulLanjewar93**
In current parse-server, I dont think there is a cap for how many objects can be updated/saved/destroyed in one go.
So maybe we should update this in docs.

### CODE REVIEW

**Review by mtrezza — COMMENTED** [review]
- What is the current batch size limit?
- Is this the only place where this limit is mentioned in the docs?

**Comment by RahulLanjewar93** [comment]
- The current batch size is 20.   `REQUEST_BATCH_SIZE: 20` https://github.com/parse-community/Parse-SDK-JS/blob/10a2e5b02db62bdb5edd996ff05198cd96d127f5/src/CoreManager.js#L187
- I am not sure , but I think batch endpoints are not mentioned in other platforms

**Comment by RahulLanjewar93** [comment]
Also, there is [this](https://github.com/parse-community/Parse-SDK-JS/pull/1090) PR on how to use `transactions` on client sdks while using the batch endpoint.
It's not merged yet and somewhere they did mention removing the batch size and putting everything in a single batch because it's causing issues.

**Comment by mtrezza** [comment]
If the current batch size is 20, not 50, should that be mentioned in the docs instead of removing the note altogether?

**Comment by RahulLanjewar93** [comment]
@mtrezza the batch size is customizable in the JS SDK. The default is set to 20.

File: _includes/rest/objects.md [hunk 1]
```diff
@@ -501,7 +501,7 @@ print(result)
 
 ## Batch Operations
 
-To reduce the amount of time spent on network round trips, you can create, update, or delete up to 50 objects in one call, using the batch endpoint.
+To reduce the amount of time spent on network round trips, you can create, update, or delete using the batch endpoint.
```
Comment (mtrezza) [inline _includes/rest/objects.md]: Thank for looking this up, should we add this?

```suggestion
To reduce the amount of time spent on network round trips, you can create, update, or delete using the batch endpoint. The batch size can be customized, the default batch size is 20.
```

Comment (RahulLanjewar93) [reply _includes/rest/objects.md]: Yes, that makes sense. I'll add that.

Comment (mtrezza) [reply _includes/rest/objects.md]: Oh you could just have commited the suggestion with the button above :-)

**Review by mtrezza — APPROVED** [review]
Looks good!

**Pull request merged**